pkg/certificate/manager.go

```diff
@@ -24,7 +24,8 @@ func NewManager(
 	ca *Certificate,
 	client client,
 	serviceCertValidityDuration time.Duration,
-	msgBroker *messaging.Broker) (*Manager, error) {
+	msgBroker *messaging.Broker,
+	certRotation chan struct{}) (*Manager, error) {
```

Comment:
To indicate this will never send anything on the channel:
```suggestion
	certRotation <-chan struct{}) (*Manager, error) {
```